hw/hw2/task1.js

```diff
@@ -0,0 +1,17 @@
+let number = 20;
```

Comment:
Better to make a function and all the variables in it 

so, should be like calcBulbsAmount and all this code in that

Reply:
fixed